projects/addon-doc/utils/sort-pages.ts

```diff
@@ -0,0 +1,41 @@
+import {TuiDocPage, TuiDocPageGroup} from '@taiga-ui/addon-doc/interfaces';
+
+import {tuiIsPageGroup} from './is-page-group';
+
+export function tuiSortPages<T extends TuiDocPage | TuiDocPageGroup>(
+    pages: readonly T[],
+    excludeSections: Set<string>,
```

Comment:
let's make the second argument optional with a default value of empty set
`excludeSection = new Set<string>()`